src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -24,6 +24,7 @@
         MPIDI_OFI_chunk_request *creq;                                  \
         MPIR_cc_incr((*sigreq)->cc_ptr, &tmp);                          \
         creq=(MPIDI_OFI_chunk_request*)MPL_malloc(sizeof(*creq), MPL_MEM_BUFFER);       \
+        MPIR_ERR_CHKANDSTMT(creq == NULL, mpi_errno, MPI_ERR_NO_MEM, goto fn_fail, "**nomem");  \
```

Comment:
This fix is fine, but the macro itself drives me a little nuts. So does its usage as part of the crazy `MPIDI_OFI_CALL_RETRY2` macro.

Reply:
Exactly. It is is used with `MPIDI_OFI_CALL_RETRY2` in `ofi_rma.h`

Reply:
Right, I get that. I'm saying the `MPIDI_OFI_CALL_RETRY2` is completely useless and obfuscating. I'm going to remove it unless someone can convince me otherwise.

Reply:
I agree. `MPIDI_OFI_retry_progress` can be called by rma sync function otherwise. @hajimefu any thoughts?

Reply:
Sounds really good to me 😃